src/Addon/Addon.php

```diff
@@ -24,7 +42,7 @@ final class rex_addon implements rex_addon_interface
     /**
      * Array of all addons.
      *
-     * @var array<non-empty-string, self>
+     * @var array<non-empty-string, Addon>
```

Comment:
Interessant, hat rector hier überall `self` ersetzt in der Datei?
Die Stellen sollten alle bei `self` bleiben.

Reply:
> Interessant, hat rector hier überall self ersetzt in der Datei?

Ja, wobei der volle Namespace erst eingetragen wurde. Ich meine CS hat es an einigen Stellen wieder angepasst, andere hatte ich dann manuell nachgezogen.

> Die Stellen sollten alle bei `self` bleiben.

Ändere ich

Reply:
Ja eigentlich ändert der CS-Fixer auf `self`. Aber der kann mit den Array-PHPDocs noch nicht so richtig umgehen.